src/unix/linux_like/mod.rs

```diff
@@ -222,6 +227,17 @@ s_no_extra_traits! {
         pub u64: u64,
     }
 
+    // Can't correctly impl Debug for unions
+    #[allow(missing_debug_implementations)]
+    pub union __c_anonymous_sigev_un {
+        #[cfg(target_pointer_width = "64")]
+        _pad: [::c_int; (64 - 2 * 4 - 8) / 4],
```

Comment:
Could you move this to a  `const SIGEV_PAD_SIZE: usize = ...` to match the source?